app/views/jobs/_new.html.haml

```diff
@@ -7,19 +7,19 @@
         = form_with scope: :job, url: jobs_path, id: :create_job do |form|
             #error_messages
             = form.label :profession, class: "form-label"
-            = form.text_field :profession, placeholder: "Profession", class: "form-input-field"
+            = form.select :profession, options_for_select(@professions), { include_blank: true }, class: "form-input-field"
             = form.label :hospital, class: "form-label"
-            = form.select :hospital, Hospital.all.collect {|c| [ c.name ] }, {include_blank: 'Please select'}, id: "Hospital", class: "form-input-field"
+            = form.select :hospital, Hospital.all.collect {|c| [ c.name ] }, { include_blank: true }, id: "Hospital", class: "form-input-field"
             = form.label :department, class: "form-label"
-            = form.text_field :department, placeholder: "Department", class: "form-input-field"
+            = form.select :department, options_for_select(@departments), { include_blank: true }, class: "form-input-field"
             = form.label :license, class: "form-label"
-            = form.select :license, License.all.collect {|c| [ c.name ] }, {include_blank: 'Please select'}, class: "form-input-field"
+            = form.select :license, License.all.collect {|c| [ c.name ] }, { include_blank: true }, class: "form-input-field"
```

Comment:
Is there an association between Job and License? A joint table?

Reply:
Not yet although one job should have one or many